zigbee-api/src/main/java/org/bubblecloud/zigbee/network/port/ZigBeeNetworkManagerImpl.java

```diff
@@ -164,6 +168,16 @@ public ZigBeeNetworkManagerImpl(ZigBeePort port, NetworkMode mode, int pan, int
             logger.trace("Using RESEND_MAX_RETRY set as DEFAULT {}", aux);
         }
         RESEND_ONLY_EXCEPTION = b;
+        
+        aux = BOOTLOADER_MAGIC_BYTE_DEFAULT;
+        try {
+            aux = Integer.parseInt(System.getProperty(BOOTLOADER_MAGIC_BYTE_KEY));
+            logger.trace("Using BOOTLOADER_MAGIC_BYTE set from enviroment {}", aux);
+        } catch (NumberFormatException ex) {
```

Comment:
As a side note it is not good style to just catch exception here. Rather one should do existence check for the system property before trying to parse it.